README.md

```diff
@@ -232,6 +232,7 @@ Keys supported by image output:
 * `registry.insecure=true`: push to insecure HTTP registry
 * `oci-mediatypes=true`: use OCI mediatypes in configuration JSON instead of Docker's
 * `unpack=true`: unpack image after creation (for use with containerd)
+* `ensure-blobs=true`: ensure blobs of the result image exist in the content store
```

Comment:
Why do we need the extra key? Don't we always need to run this when exporting images to containerd?

Reply:
Thank you for the comment. This is because tests require blobs being lazy even after exporting it with `type=image,push=true` https://github.com/moby/buildkit/runs/6000088966?check_suite_focus=true

Reply:
If it creates a broken image in containerd then it looks wrong. Maybe we need an option that would allow to push the image without adding it to the containerd image store. In that case blobs could remain lazy.

Reply:
> an option that would allow to push the image without adding it

It won't pass some tests like `testLazyImagePush` which performs the check using lazy images in the image store.
How about making `ensure-blobs=true` the default behaviour when a client creates an image in the image store?

Reply:
Maybe better would be `store=false` that would skip saving image to the image store.

Reply:
Added `store` option. I think this should be `store=true` by default for keeping the backward-compatibility of `type=image` behaviour.

https://github.com/moby/buildkit#containerd-image-store

> The containerd worker needs to be used
> buildctl build ... --output type=image,name=docker.io/username/image
> ctr --namespace=buildkit images ls